extensions/nextjs-authkit/README.md

```
@@ -0,0 +1,145 @@
+3. Under **Redirects**, set:
+   - **Redirect URI**: `http://localhost:3000/callback` for local dev
+   - **Sign-in endpoint**: `http://localhost:3000/sign-in`
+   - **Logout URI**: wherever you want users sent after sign-out
+4. (Optional) Configure MFA, password policy, and SSO connections under
+   **Authentication** and **Organizations**.
+
+### 2. Environment configuration
+
+Copy the appended variables from `.env.example` into `.env.local` and fill them in:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)